src/Sarif/ExtensionMethods.cs

```diff
@@ -182,56 +182,60 @@ public static string GetMessageText(this Result result, IRule rule, bool concise
             if (string.IsNullOrEmpty(text))
             {
                 text = string.Empty;    // Ensure that it's not null.
-
+                
                 string messageId = result.Message?.MessageId;
+                string[] arguments = null;
 
-                if (rule != null && !string.IsNullOrWhiteSpace(messageId))
+                if (result.Message?.Arguments != null)
+                {
+                    arguments = new string[result.Message.Arguments.Count];
+                    result.Message.Arguments.CopyTo(arguments, 0);
+                }
+                else
                 {
-                    string messageString;
+                    arguments = new string[0];
+                }
 
-                    string[] arguments = null;
+                if (rule != null
+                    && !string.IsNullOrWhiteSpace(messageId)
+                    && rule.MessageStrings?.ContainsKey(messageId) == true)
+                {
+                    string formatString = rule.MessageStrings[messageId];
+                    text = GetFormattedMessage(formatString, arguments);
+                }
+            }
 
-                    if (result.Message?.Arguments != null)
-                    {
-                        arguments = new string[result.Message.Arguments.Count];
-                        result.Message.Arguments.CopyTo(arguments, 0);
-                    }
-                    else
-                    {
-                        arguments = new string[0];
-                    }
+            if (concise)
+            {
+                text = GetFirstSentence(text);
+            }
 
-                    if (rule.MessageStrings?.ContainsKey(messageId) == true)
-                    {
-                        messageString = rule.MessageStrings[messageId];
+            return text;
+        }
+
+        internal static string GetFormattedMessage(string formatString, string[] arguments)
+        {
+            string result = null;
```

Comment:
>result [](start = 19, length = 6)

Consider `result` => `formattedMessage`

Reply:
Bad habit

---
In reply to: [212704164](https://github.com/Microsoft/sarif-sdk/pull/1048#discussion_r212704164) [](ancestors = 212704164)